### ISSUE TITLE
Worlds should have a visual saved/save failed/needs save indicator

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#103]
Similar to the red rimmed screen in the OE.  When I save a world, I need to watch for the popup to show up in the bottom right to know whether it worked or not; what I'd like is an indicator that is more persistent and where any failure is easier to trace.

**Comment by merryman** [comment]
I agree, however in principle that indiciator would basically be always on "needs save" or "save crashed". Since also moving the mouse cursor itself has effects on the state of the world. (Hand morph being manipulated).
I would at least add to that a policy of which changes to track for and which not.
Ideally we would want the system to continously store changes whenever source changes or changes to the morph hierarchy happens.
More thought needs to be put into this. Maybe this is a nice addition to once we have figured out acyclic representation of lively.

**Comment by linusha** [comment]
> Ideally we would want the system to continously store changes ~~whenever source changes or~~ changes to the morph hierarchy happens.

We can implement this for the new "workspaces" which will come with the new project management tooling.
